src/api/definition.ts

```diff
@@ -147,16 +148,16 @@ export class ApiDefinition extends apigateway.ApiDefinition {
     }
 
     // Loop through paths to ensure all paths are defined in OpenAPI schema
-    Object.keys(paths).forEach(path => {
+    Object.keys(paths || {}).forEach(path => {
```

Comment:
probably makes sense to add the default value in the configurePaths params:
```ts
private configurePaths(paths: Paths = {}
```

If possible.